.dockstore.yml

```diff
@@ -198,23 +193,20 @@ workflows:
        branches:
          - master
          - ah_var_store
-         - gg_AddPrecisionAndSensitivityWdl
    - name: GvsQuickstartIntegration
      subclass: WDL
      primaryDescriptorPath: /scripts/variantstore/wdl/GvsQuickstartIntegration.wdl
      filters:
        branches:
          - master
          - ah_var_store
-         - vs_357_quickstart_integration
    - name: GvsIngestTieout
      subclass: WDL
      primaryDescriptorPath: /scripts/variantstore/wdl/GvsIngestTieout.wdl
      filters:
        branches:
          - master
          - ah_var_store
-         - vs_261_ingest_errors
```

Comment:
if I had looked at this pr earlier, I would have seen that you already did this!